spec/db.spec.js

```diff
@@ -144,6 +151,41 @@ describe('db', () => {
           });
         });
 
+        describe('.getTableCreateScript', () => {
+          it('should return table create script', async() => {
+            const [createScript] = await dbConn.getTableCreateScript('users');
+
+            if (dbClient === 'mysql') {
+              expect(createScript).to.contain('CREATE TABLE `users` (\n' +
```

Comment:
Since we are using ES6. Couldn't you use template string instead of these concatenations? Template string keeps the code more readable.